aten/src/ATen/cuda/CUDASolver.cpp

```diff
@@ -1390,6 +1390,40 @@ void xgeqrf<c10::complex<double>>(CUDASOLVER_XGEQRF_ARGTYPES(c10::complex<double
       info));
 }
 
+void xgetrf_buffersize(CUDASOLVER_XGETRF_BUFFERSIZE_ARGTYPES) {
```

Comment:
Since using 64-bit interface is not part of this PR, maybe we shouldn't include the `xgetrf`, that is not yet used, either?